src/apps/assets/avatars/components/SingleAvatar/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { useQuery } from '@tanstack/react-query';
+import { useParams } from '@tanstack/react-router';
+import { Alert, Card, Descriptions, Image, Spin, Typography } from 'antd';
+import dayjs from 'dayjs';
+
+import { avatarQueries } from '@/shared/service/query/avatar';
+
+export default function SingleAvatar() {
+  const { id: avatarId } = useParams({ from: '/(menus)/assets/avatars/$id/' });
+  const {
+    data: avatar,
+    isLoading,
+    isError,
+  } = useQuery({
+    ...avatarQueries.getSingle(avatarId!),
```

Comment:
```suggestion
    ...avatarQueries.getSingle(avatarId),
```
id param이 스키마상 옵셔널 필드인가요? 이 assertion이 필요없을거 같아서유